.github/workflows/test-all.yml

```diff
@@ -2253,6 +2253,25 @@ jobs:
       test-py3: ${{ inputs.test-py3 }}
       minimum-base-package: ${{ inputs.minimum-base-package }}
     secrets: inherit
+  j74dc677:
+    uses: ./.github/workflows/test-target.yml
+    with:
+      job-name: Nvidia Triton
+      target: nvidia_triton
+      platform: linux
+      runner: '["ubuntu-22.04"]'
+      repo: "${{ inputs.repo }}"
+      python-version: "${{ inputs.python-version }}"
+      standard: ${{ inputs.standard }}
+      latest: ${{ inputs.latest }}
+      agent-image: "${{ inputs.agent-image }}"
+      agent-image-py2: "${{ inputs.agent-image-py2 }}"
+      agent-image-windows: "${{ inputs.agent-image-windows }}"
+      agent-image-windows-py2: "${{ inputs.agent-image-windows-py2 }}"
+      test-py2: ${{ inputs.test-py2 }}
```

Comment:
Do we intend to test this on python2 as the integration is for python3.9+ ?

Reply:
It won't be tested with Python 2 but this is part of the default template for every single integration. On Py2 jobs, the integration will just get skipped

Hopefully we'll be able to simplify this and drop the Py2 part soon 🤞